firedrake/slate/slac/compiler.py

```diff
@@ -224,7 +242,7 @@ def generate_kernel_ast(builder, statements, declared_temps):
     # Now we wrap up the kernel ast as a PyOP2 kernel and include the
     # Eigen header files
     include_dirs = builder.include_dirs
-    include_dirs.extend(["%s/include/eigen3/" % d for d in PETSC_DIR])
+    include_dirs.append(EIGEN_INCLUDE_DIR)
```

Comment:
This was already the case, but while we're here, can we fix this.
Say:
```python
include_dirs = list(builder.include_dirs)
```
So that a builder if reused doesn't make a longer-and-longer list of include_dirs

Reply:
Thanks.